backend/api/migrations/0009_organization_edrms_record.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.20 on 2023-10-10 22:21
+
+from django.db import migrations, models
+
+def change_organization_types(apps, schema_editor):
```

Comment:
Is this supposed to be here? I don't see this as part of ticket #2579

Reply:
if you look at the wireframe, supplier Type radio button label changed.
https://preview.uxpin.com/59145260c7d8cc48969ac79401cc03d84843c3d9#/pages/161516664/simulate/no-panels?mode=mf